### PR TITLE
Portfolio main page

### PR DESCRIPTION
**Pull request by Anwaar12**
I created a new GitHub repo
I created a new branch according to the [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) rules.
[I Added a .gitignore file.
I Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.
I added a descriptive README file to my project
I Added3 files (each one in a separate commit:
I created A .html file that includes information about me and link to my social media accounts.I put picture to some of my project which I made and also links to it and after that I put information about my development career.
I created A .css file that includes styles to the icons of the social media apps. In this, I use flexbox to position elements and add styles to logos and other content. I use grid to place element on page.
I added A test.md file with the text "This file should be ignored by git".
I made sure that the last file is actually ignored by git and is not present in your GitHub repository.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          <li><img src="./images/gmail.svg" alt="gmail" /></li>
+          <li>
+            <img src="./images/github.svg" alt="github" />
+          </li>
+          <li>
+            <img src="./images/Vector (3).svg" alt="peace" />
+          </li>
+        </ul>
+      </section>
+    </section>
```
Comment (cynthiainga) [inline index.html]: - [ ] I think all these parts have only two sections, and for good practice, I think it;s not better to put section tags inside the section tag. The section tag it represents a generic standalone section of the document. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info 🙃 🙃

File: index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  </head>
+  <body>
+    <section class="container">
+      <section class="header">
+        <nav>
+          <h3 class="mylogo">My Logo</h3>
+          <div class="menu-bars"></div>
+          <span class="material-symbols-outlined"> menu </span>
+        </nav>
+      </section>
```
Comment (cynthiainga) [inline index.html]: - [ ] The container of this section is a `header` tag, the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info 😉 😉
- [ ] Good job by implementing the logo but the logo is on the left and has a link (`a` tag). This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info 🙃
- [ ] The hamburger button has a `nav` tag and not all the section, please kindly fix it 🙃 🙃

File: index.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            <img src="./images/linkedin1.png" alt="linkedin" />
+            <img src="./images/linkedin2.png" alt="linkedin" />
+          </li>
+          <li><img src="./images/gmail.svg" alt="gmail" /></li>
+          <li>
+            <img src="./images/github.svg" alt="github" />
+          </li>
+          <li>
+            <img src="./images/Vector (3).svg" alt="peace" />
+          </li>
```
Comment (cynthiainga) [inline index.html]: - [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag), the `a` tag will let you connect your social networks with the icons, using href 😊 😉

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+.main-content {
+  width: 375px;
+  height: 718px;
+  background-color: #fff;
+  border-radius: 1px 1px 1px 40px;
+  background-image: url("./images/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+}
```
Comment (cynthiainga) [inline styles.css]: - [ ] Please kindly make sure your design sticks as much as possible to the Figma design, the background image has a border-radius at the left, please kindly fix it 🙃 😉 

Your design

Screen Shot 2022-09-27 at 17 19 59

Figma design

Screen Shot 2022-09-27 at 17 20 39

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Anwaar12, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Used correct Github flow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- The commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8) don't forget it in your next commit 😉 🙃 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sajid-munawar — APPROVED** [review]
### Hi @Anwaar12  , 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
 ## [Optional] suggestions:
 * [ ]  Please check my inline comments.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @sajid-muanwar  in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tugsem** [comment]
Hi @Anwaar12 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a Pull Request Link, that belongs to the previous project. You should merge and close your pull requests after the project is approved. After that, start working on a new branch for new features.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**